lib/Infer/AliveDriver.cpp

```diff
@@ -530,12 +536,21 @@ std::string getUniqueName() {
 bool souper::AliveDriver::translateAndCache(const souper::Inst *I,
                                             IR::Function &F,
                                             Cache &ExprCache) {
+  if (souper::Inst::isOverflowIntrinsicSub(I->K)) {
+    return true; // Unused translation
+  }
+
   if (ExprCache.find(I) != ExprCache.end()) {
     return true; // Already translated
   }
 
+  auto Ops = I->Ops;
+  if (souper::Inst::isOverflowIntrinsicMain(I->K)) {
+    Ops = Ops[0]->Ops;
+  }
+
   FunctionBuilder Builder(F);
-  for (auto &&Op : I->Ops) {
+  for (auto Op : Ops) {
```

Comment:
plz explain why you chose to make a copy of each op instead of using rvalue references

Reply:
fixed it.